models/api_process_shipment.go

```diff
@@ -21,6 +21,7 @@ type Shipment struct {
 
 	// Only used for international ground shipments
 	OriginatorName    string
+	TotalWeight       Weight
```

Comment:
Just double checking if total weight is used for international only?

Reply:
I'll remove old comments that longer apply